backend/plugin/db/spanner/sync.go

```diff
@@ -90,7 +89,6 @@ func (d *Driver) SyncDBSchema(ctx context.Context) (*storepb.DatabaseSchemaMetad
 	for schemaName := range schemaNameMap {
 		schemaNames = append(schemaNames, schemaName)
 	}
-	sort.Strings(schemaNames)
```

Comment:
Since we're going through a map, the key ranges are unstable.

Reply:
done